src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,89 @@
-import { useState } from 'react';
+import { FC, FormEventHandler, useState } from 'react';
 import { TextField } from '../TextField';
+import { Movie } from '../../types/Movie';
 
-export const NewMovie = () => {
-  // Increase the count after successful form submission
-  // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+interface Props {
+  onAdd: (newMovie: Movie) => void
+}
+
+export const NewMovie:FC<Props> = ({ onAdd }) => {
+  const [count, setCount] = useState(0);
+
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [imgUrl, setImgUrl] = useState('');
+  const [imdbUrl, setImdbUrl] = useState('');
+  const [imdbId, setImdbId] = useState('');
+
```

Comment:
You can use only one useState with object here and set its value in onChange, it looks more clear and simple i think, but you can use your variant, it's just suggestion :)

```suggestion
export const NewMovie:FC<Props> = ({ onAdd }) => {
  const [count, setCount] = useState(0);

  const [newMovie, setMovie] = useState({
    title: '',
    description: '',
    imgUrl: '',
    imdbUrl: '',
    imdbId: '',
  });

```